enterprise/utils.py

```diff
@@ -1962,3 +2069,28 @@ def get_best_mode_from_course_key(course_key):
         else CourseModes.AUDIT
 
     return best_course_mode
+
+
+def parse_lms_api_datetime(datetime_string, datetime_format=LMS_API_DATETIME_FORMAT):
```

Comment:
I moved it here to utils since calling EnterpriseCatalogApiClient  from tasks was causing a circular dep. I think this is a better location for this function anyway